master/internal/stream/metrics.go

```diff
@@ -28,6 +29,7 @@ type MetricMsg struct {
 	bun.BaseModel `bun:"table:metrics"`
 
 	// immutable attributes
+	// XXX (corban): if this gets removed, what happens to delete msg?
```

Comment:
Yep, on second thought, since metrics are intended to be append-only, it doesn't make sense to even have an implemented DeleteMsg; instead it should be a noop/return an error internally if there's an attempt to call it.